src/RayMarchedAtlasVolume.ts

```diff
@@ -366,4 +367,8 @@ export default class RayMarchedAtlasVolume {
     this.setUniform("textureAtlas", this.channelData.getFusedTexture());
     this.setUniform("textureAtlasMask", this.channelData.maskTexture);
   }
+
+  public setRenderUpdateListener(_listener?: ((iteration: number) => void) | undefined) {
+    0;
```

Comment:
Any reason for this statement vs. a return?

Reply:
No special reason. As I recall, typescript didn't want it to be just empty, and putting a 0 seemed like the least amount of typing to have the function do nothing. :).  I know things like lodash provide a `noop` but I figured this did the job without bringing a dependency.  Now that you mention this, `return;` seems a bit more obvious.